drivers/scheduler/k8s/specs/nginx/px-nginx-storage.yaml

```diff
@@ -11,12 +11,10 @@ kind: PersistentVolumeClaim
 apiVersion: v1
 metadata:
   name: px-nginx-pvc
-  annotations:
-    azure-storage-class: azure-file-sc-nginx
 spec:
-  storageClassName: px-nginx-sc
+  storageClassName: nginx-sc
   accessModes:
-    - ReadWriteMany
+    - ReadWriteOnce
```

Comment:
why this change?

For nginx, the PVC needs to be ReadWriteMany.

Reply:
Well, since aws-ebs does not support read write many we need to have least common denomitor for this or not to use this app for EKS tests untill aws bring provisioner with RWM support.